### ISSUE TITLE
Assertion keyword has impact on the next one called in sequence

### ISSUE THREAD

**Issue by Tset-Noitamotua** [asyrjasalo/RESTinstance#73]
![image](https://user-images.githubusercontent.com/5635462/59964179-7c9fe080-94fd-11e9-9717-7047b1b18ef8.png)

Then called in sequence the first keyword has an impact on the second.

```robotframework
String   response body 0 templateId  minimal_instruction.en.v1
Array    response body
```

**Question**: Is this intended?

**Comment by asimell** [comment]
This is not intended behaviour. We'll take a look if we can reproduce this and fix it.

**Comment by Atihinen** [comment]
Couldn't reproduce with this one https://github.com/eficode/RESTinstance/blob/issue_73/atest/validations.robot#L25

**Comment by asimell** [comment]
I managed to reproduce this and looks like [this](https://github.com/eficode/RESTinstance/commit/9d6a47d99ad977f3bdb5a07d0fbc2307d7556f57) fixes the issue. We'll create a PR for that to the [Eficode fork](https://github.com/eficode/RESTinstance) once we've written proper unit tests and then create a release PR to this repository.

**Comment by asimell** [comment]
@Tset-Noitamotua As a workaround you can use `skip=true` as the last argument in your `Array` keyword to prevent it validating the schema.

I'm still working on an actual solution for this.

**Comment by asimell** [comment]
I created https://github.com/eficode/RESTinstance/pull/19 to fix this issue. The problem was `self.request["auth"]` which prevented the use of `deepcopy`.

**Comment by asimell** [comment]
PR is merged. The fix will be available in 1.3.0 release, which we aim to do at the beginning of February. Closing this issue for now.
